Shopify.md

```diff
@@ -0,0 +1,19 @@
+# BTCPay Server and Shopify Integration
+
+To integrate BTCPay Server into Shopify, you need to be running BTCPay Server v1.0.5.5 or newer.
+
+* In your BTCPay Server Store > Pay button enable `Allow anyone to create invoice`
+* In Shopify Settings > Payment Providers > Manual Payment Methods add one which contains "Bitcoin with BTCPayServer"
+* In Shopify Settings > Checkout > Additional Scripts input the following script, with the details from your BTCPayServer instead of the placeholder values.
```

Comment:
Won't be needed anymore as long as the private app is created with a "Script tags -Read and Write" permission

Reply:
If the permission is not provided (or you are testing on localhost and it does not allow that), this will show which also links you directly to the textbox to paste that script in
![image](https://user-images.githubusercontent.com/1818366/93619143-7803f980-f9d8-11ea-988b-c9fdcb909111.png)